src/navtypes.h

```diff
@@ -14,6 +14,12 @@ struct gpscoords_t {
 	double lon;
 };
 
+struct eulerangles_t {
+	double roll;
+	double pitch;
+	double yaw;
+};
```

Comment:
Even though this is self-explanatory, can we put doxy comments here? Could be useful to link here https://en.wikipedia.org/wiki/Euler_angles#Tait%E2%80%93Bryan_angles and mention that the rotation ordering is XYZ extrinsic rotations.